### PR TITLE
Add GYM-5028 content

### PR DESCRIPTION
**Pull request by jgagne**
## This PR adds Take 5 GYM-5028 content:

- Front matter: `_data/take5s/GYM-5028.yml`
- Meta stub: `take5/meta/gym-5028.md`
- Transcript: `take5/gym-5028.md`
- Poster and OG artwork: `img/take5/posters/gym-5028.jpg`, `img/take5/posters/gym-5028-hero.jpg`, and `img/take5/og/gym-5028-og.png`
- - -
- Closes: https://github.com/gymnasium/tracker/issues/24

### CODE REVIEW

File: _data/take5s/GYM-5028.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+course_ID: GYM-5028
+title: "Adding a Dark Mode Theme to Your Website"
+date: 2020-02-17T00:00:00-04:00
+course_type: take5
+url: /static/take5/GYM-5028/
+poster_art: /img/take5/posters/gym-5028.jpg
+live: false
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5028.yml]: @jgagne - don't you want this to be `live: true`?

File: _data/take5s/GYM-5028.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+instructor: "Jeremy Osborn"
+topic: "Web Design & Development"
+video_ID: VYhPXZvAD9k
+video_duration: "4:57"
+featured: false
```
Comment (andrewpmiller) [inline _data/take5s/GYM-5028.yml]: @jgagne - same question here. Since we'll have this live and featured on Monday afternoon...

**Review by andrewpmiller — COMMENTED** [review]
Everything looks good to me. Just wondering about setting the live/featured so it's a simple staging ▶️  production merge on Monday (see my questions on GYM-5028.yml)

File: _data/take5s/GYM-5028.yml (hunk 1 above)
Comment (jgagne) [reply _data/take5s/GYM-5028.yml]: Indeed, I'll update.

**Comment by jgagne** [comment]
Prepped for release.

**Review by andrewpmiller — APPROVED** [review]

**Pull request merged**